fintoc/mixins/resource_mixin.py

```diff
@@ -30,7 +30,8 @@ def __init__(self, client, handlers, methods, path, **kwargs):
                 klass = get_resource_class(resource, value=value)
                 setattr(self, key, objetize(klass, client, value))
             else:
-                setattr(self, key, value)
+                klass = get_resource_class(resource, value=value)
+                setattr(self, key, objetize(klass, client, value))
```

Comment:
I think we can remove the last `else`? The code looks the same

Reply:
You're right!